daffodil-test/src/test/resources/org/apache/daffodil/usertests/SepTests.tdml

```diff
@@ -278,15 +274,11 @@
 
     <xs:include schemaLocation="org/apache/daffodil/xsd/DFDLGeneralFormat.dfdl.xsd"/>
     <dfdl:format
-      ref="ex:GeneralFormatPortable"
-      representation="text"
-      lengthKind="delimited"
-      separatorPosition="infix"
-      dfdlx:emptyElementParsePolicy="treatAsMissing"/> <!-- remove extension proerty for IBM cross tests -->
-    <!--
-    Note: dfdlx:emptyElementParsePolicy should become regular DFDL emptyElementParsePolicy
-    once implemented in DAFFODIL-2496. The enum 'treatAsMissing' is renamed to 'treatAsAbsent'
-    -->
+            ref="ex:GeneralFormatPortable"
+            representation="text"
+            lengthKind="delimited"
+            separatorPosition="infix"
+            dfdlx:emptyElementParsePolicy="treatAsMissing"/> <!-- remove extension property for IBM cross tests -->
```

Comment:
Should this be just no-prefix emptyElementParsePolicy='treatAsAbsent' now?

I probably wrote this test originally, but I don't understand the comment "remove extension property for IBM cross tests" since this is used by an IBM cross test below. It must be that IBM DFDL is able to ignore the dfdlx extensions or this wouldn't have worked in test test_sep_ssp_never_4_ibm.

So I think the comment line "remove extension property for IBM cross tests" is spurious now.

Reply:
Updated during our meeting.